### PR TITLE
[zh-cn]: translation for Window.cookieStore property

### PR DESCRIPTION
**Pull request by xyy94813**
### Description

translation to zh-CN for Window.cookieStore property

### Motivation

Learning and translate.

### Additional details

None.

### Related issues and pull requests

None.

### CODE REVIEW

File: files/zh-cn/web/api/window/cookiestore/index.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: Window：cookieStore 属性
+slug: Web/API/Window/cookieStore
+---
+
+{{securecontext_header}}{{APIRef("Cookie Store API")}}{{SeeCompatTable}}
+
+{{domxref("Window")}} 接口的 **`cookieStore`** 只读属性，返回当前文档上下文的 {{domxref("CookieStore")}} 对象的引用。是 [Cookie Store API](/zh-CN/docs/Web/API/Cookie_Store_API) 的入口点。
```
Comment (jasonren0403) [inline files/zh-cn/web/api/window/cookiestore/index.md:8]:
```suggestion
{{domxref("Window")}} 接口的 **`cookieStore`** 只读属性返回当前文档上下文的 {{domxref("CookieStore")}} 对象的引用，是 [Cookie Store API](/zh-CN/docs/Web/API/Cookie_Store_API) 的入口点。
```

File: files/zh-cn/web/api/window/cookiestore/index.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+title: Window：cookieStore 属性
+slug: Web/API/Window/cookieStore
```
Comment (jasonren0403) [inline files/zh-cn/web/api/window/cookiestore/index.md:3]: 可以考虑加上l10n.sourceCommit属性，属性值为当前对应英文文档的最新提交SHA值

**Review by jasonren0403 — COMMENTED** [review]
一些小问题

File: files/zh-cn/web/api/window/cookiestore/index.md (hunk 2 above)
Comment (xyy94813) [inline files/zh-cn/web/api/window/cookiestore/index.md:3]:
```suggestion
slug: Web/API/Window/cookieStore
l10n:
  sourceCommit: fb6d05866d57baf66770dfa30bc88b7b967cba9e
```

Comment (xyy94813) [reply files/zh-cn/web/api/window/cookiestore/index.md:3]:
```suggestion
slug: Web/API/Window/cookieStore
l10n:
  sourceCommit: fb6d05866d57baf66770dfa30bc88b7b967cba9e
```

**Review by jasonren0403 — APPROVED** [review]
👍 感谢贡献，欢迎来到MDN！

**Pull request merged**